Update/zwata_002_vm00_n02.txt

```diff
@@ -62,7 +62,7 @@ void dialog004()
 	// (backup) SetValidityOfInput( TRUE );
 
 	OutputLine(NULL, "……ぱ、ぱんつまで女の子のだって…バ、バレちゃうよ？」",
-		   NULL, " find out that I'm wearing women's clothing right down to the unmentionables?\"", GetGlobalFlag(GLinemodeSp));
+		   NULL, " find out that you're wearing women's clothing right down to the unmentionables.\"", GetGlobalFlag(GLinemodeSp));
```

Comment:
Hmm japanese has a question mark here. I think I'll get Norgus to check this later.

Reply:
the meaning of the sentence in english is now an affirmation, so I don't think a question mark is appropriate.

Reply:
@Norgus Alright. The change from I'm to you're is correct too, right?

Reply:
I ported the change Norgus had made of when I reported it. Because else it would have reverted it, I think?

Reply:
I thought it looked like the change I made! It was in response to a raised issue.

Reply:
> Because else it would have reverted it, I think?

It would not. The changes would be merged together.